lib/unsupported-features/node-builtins-modules/worker_threads.js

```diff
@@ -16,6 +16,7 @@ const worker_threads = {
             supported: ["17.5.0", "16.15.0"],
         },
     },
+    markAsUncloneable: { [CALL]: { supported: ["23.0.0", "22.10.0"] } },
```

Comment:
Same here: `markAsUncloneable` is a function, hence using `CALL`.
However, functions like `markAsUntransferable` are using `READ` (incorrectly I presume?)